api/v1/kataconfig_types.go

```diff
@@ -26,6 +26,7 @@ type KataConfigSpec struct {
 	// KataConfigPoolSelector is used to filer the worker nodes
 	// if not specified, all worker nodes are selected
 	// +optional
+	// +nullable
```

Comment:
what does this do?

Reply:
The validation now checks if it's nullable or not.